WcaOnRails/app/models/regulation.rb

```diff
@@ -0,0 +1,21 @@
+class Regulation < SimpleDelegator
+  begin
+    REGULATIONS = JSON.parse(File.read(Rails.root.to_s + "/app/views/regulations/wca-regulations.json")).freeze
+  rescue
+    REGULATIONS = [].freeze
```

Comment:
Ahah just remember it makes travis fail because it doesn't build the regulations :)